### PR TITLE
chore: instrument ingestor workflow

### PR DESCRIPTION
**Pull request by wa0x6e**
This PR add a instrument the ingestor function

## Changes

- Wrap the whole ingestor content inside a try/finally
- Move some variable definition outside the try block (so we can access them in finally)
- Start a timer, and end it in the `finally` block

### CODE REVIEW

File: src/ingestor.ts [hunk 1]
```diff
@@ -12,208 +12,218 @@ import { storeMsg } from './helpers/highlight';
...
 
 const NAME = 'snapshot';
 const VERSION = '0.1.4';
 const broviderUrl = process.env.BROVIDER_URL || 'https://rpc.snapshot.org';
 
 export default async function ingestor(req) {
-  const body = req.body;
+  const success = 0;
+  const type = '';
+  const network = '1';
```
Comment (ChaituVR) [inline src/ingestor.ts:24]: We are not changing this anywhere?

**Review by ChaituVR — APPROVED** [review]
utAck

File: src/ingestor.ts [hunk 2]
```diff
@@ -12,208 +12,218 @@ import { storeMsg } from './helpers/highlight';
...
+import { timeIngestorProcess } from './helpers/metrics';
 
 const NAME = 'snapshot';
 const VERSION = '0.1.4';
 const broviderUrl = process.env.BROVIDER_URL || 'https://rpc.snapshot.org';
 
 export default async function ingestor(req) {
-  const body = req.body;
+  const success = 0;
+  const type = '';
```
Comment (ChaituVR) [inline src/ingestor.ts:23]: We are not changing these anywhere? changes automatically?

Comment (wa0x6e) [reply src/ingestor.ts]: fixed, merge conflict gone wrong

**Pull request merged**